### PR TITLE
feat(experimental): new component `Cell`

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the new behavior?
[Figma](https://www.figma.com/file/E5smFo7uxLRjWx7Ov2e3zt/tui-cell?type=design&node-id=70-80388&mode=design&t=yCzrWPW3lQoshGfh-0)

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/experimental/components/cell/cell.component.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    styleUrls: ['./cell.styles.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        '[attr.data-size]': 'size',
+        '[attr.data-platform]': 'platform',
+    },
+})
+export class TuiCellComponent {
+    @Input()
+    title: PolymorpheusContent = '';
```
Comment (vladimirpotekhin) [inline projects/experimental/components/cell/cell.component.ts:18]: It seems like sort of layout component, should we use content projection with `tuiSlot` for this type of component to unify our API? should we make some rules about this?

what do you think @waterplea @splincode @nsbarsukov ?

Comment (nsbarsukov) [reply projects/experimental/components/cell/cell.component.ts:18]: fixed ✅

Comment (nsbarsukov) [reply projects/experimental/components/cell/cell.component.ts]: fixed ✅

**Comment by vladimirpotekhin** [comment]
something wrong with the s-size icon
https://taiga-ui--pr5460-cell-6chs8fss.web.app/experimental/cell/API?size=s

Снимок экрана 2023-10-16 в 14 40 06

File: projects/demo/src/modules/experimental/cell/examples/3/index.less [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[tuiCell] {
+    max-width: 25rem;
+    white-space: nowrap;
+
+    // You can customize shrink ratio of sides by this variables
+    --tui-left-shrink: 4;
+    --tui-right-shrink: 5;
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/experimental/cell/examples/3/index.less:7]: nice 👍

**Comment by nsbarsukov** [comment]
> something wrong with the s-size icon https://taiga-ui--pr5460-cell-6chs8fss.web.app/experimental/cell/API?size=s

@vladimirpotekhin fixed

File: projects/demo/src/modules/experimental/cell/examples/3/index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            +$30
+        </tui-badge>
+        <div tuiSubtitle>&nbsp;</div>
+    </div>
+</div>
+
+<div tuiCell>
+    <tui-avatar-stack>
+        <tui-avatar src="https://f4.bcbits.com/img/a0405229176_16.jpg"></tui-avatar>
+        <tui-avatar src="https://f4.bcbits.com/img/a1487194314_16.jpg"></tui-avatar>
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/experimental/cell/examples/3/index.html:27]: images are not loading

File: projects/demo/src/modules/experimental/cell/examples/import/insert-template.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+```html
+<div tuiCell>
+  <span tuiSlot="title">Title</span>
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/experimental/cell/examples/import/insert-template.md:3]: old API

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**